### PR TITLE
after stand up meeting june 8th

### PR DESCRIPTION
**Pull request by romcar**
got the list to dynamically render the list data 

persisted data that gets updated in the list

### CODE REVIEW

**Comment by JohnWebb4** [comment]
lgtm

File: database/index.js [hunk 1]
```diff
@@ -34,44 +34,27 @@ var saveUser = function(user) {
...
-      })
-    }
-  }).then(listExists => {
-    List.findOneAndUpdate({name: listExists.name}, { "$set": {"items": newList.items, "name": newList.name, "user_id": newList.user_id, "total_price": newList.total_price} }, {new: true}, function(err, doc) {
-      if (err) return res.end(500, {error: err});
-      res.end('Updated existing list');
-    })
-  }).catch(err => console.error(err));
-}
+// var updateList = function(list) {
```
Comment (romcar) [inline database/index.js]: this function will be removed

**Comment by mealeark** [comment]
lgtm

**Comment by romcar** [comment]
closes #126 
closes #127
closes #131
closes #123

**Pull request merged**